### PR TITLE
Add deserializing reset command as JSON object to Sensor Simulator

### PR DESCRIPTION
**Pull request by xitia**
Added deserializing reset command as JSON object to sensor simulator

### CODE REVIEW

File: edge-modules/SimulatedTemperatureSensor/src/Program.cs [hunk 1]
```diff
@@ -126,10 +127,24 @@ static Task<MessageResponse> ControlMessageHandle(Message message, object userCo
...
                     else
                     {
-                        //NoOp
+                        // NoOp
                     }
                 }
             }
+            catch (JsonSerializationException)
+            {
+                var messageBody = JsonConvert.DeserializeObject<ControlCommand>(messageString);
```
Comment (dsajanice) [inline edge-modules/SimulatedTemperatureSensor/src/Program.cs]: Curious what scenario this supports.

Comment (xitia) [reply edge-modules/SimulatedTemperatureSensor/src/Program.cs]: This is for tutorial with ASA module, we recently updated the event output format to single JSON event (or multiple JSON events delimited by ",").

Comment (dsajanice) [reply edge-modules/SimulatedTemperatureSensor/src/Program.cs]: Is the scenario with the array of messages still valid?

Comment (xitia) [reply edge-modules/SimulatedTemperatureSensor/src/Program.cs]: not being used by current ASA module. keep it for backward compatible

**Comment by varunpuranik** [comment]
Closing in favor of https://github.com/Azure/iotedge/pull/153

**Pull request closed without merging**